src/core_atmosphere/dynamics/mpas_atm_time_integration.F

```diff
@@ -1730,39 +1730,60 @@ subroutine atm_compute_moist_coefficients( dims, state, diag, mesh, &
       call mpas_pool_get_array(diag, 'cqw', cqw)
       call mpas_pool_get_array(diag, 'cqu', cqu)
 
+      MPAS_ACC_TIMER_START('atm_compute_moist_coefficients [ACC_data_xfer]')
+      !$acc enter data copyin(qtot, cqw, cqu, scalars)
+      MPAS_ACC_TIMER_STOP('atm_compute_moist_coefficients [ACC_data_xfer]')
+
+
+      !$acc parallel
+      !$acc loop
```

Comment:
I think the loop directives should be more explicit (e.g. `loop gang worker` here). Just to match with other work we're doing.